test/powershell/Language/Scripting/ActionPreference.Tests.ps1

```diff
@@ -420,7 +420,7 @@ Describe 'ActionPreference.Break tests' -tag 'CI' {
         }
```

Comment:
@PaulHigin, your last commit had 1 failures in `PowerShell-CI-windows`
Enter-PSHostProcess tests.By Process Id.Can enter using NamedPipeConnectionInfo
```powershell
Expected 4152, because Exception: Cannot perform the operation the runspace pool is not in the 'Opened' state. The current state is 'Broken'.; Cannot perform the operation the runspace pool is not in the 'Opened' state. The current state is 'Broken'.; Cannot perform the operation the runspace pool is not in the 'Opened' state. The current state is 'Broken'.; Cannot perform the operation the runspace pool is not in the 'Opened' state. The current state is 'Broken'.; Cannot perform the operation the runspace pool is not in the 'Opened' state. The current state is 'Broken'.;, but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Enter-PSHostProcess.Tests.ps1: line 153
153:                 $result | Should -Be $pwshId -Because $errorMsg
```

Reply:
@PoshChan Please restart Windows